analyzers/tests/SonarAnalyzer.TestFramework/Verification/IssueValidation/IssueLocationCollector.cs

```diff
@@ -119,103 +117,93 @@ public static IList<IssueLocation> MergeLocations(IssueLocation[] locations, Lis
             return Enumerable.Empty<IssueLocation>();
         }
 
+        // ToDo: Throw away
         private static IEnumerable<IssueLocation> GetBuildErrorsLocations(string filePath, TextLine line) =>
-            GetLocations(filePath, line, RxBuildError);
+            FindLocations(filePath, line, RxBuildError);
```

Comment:
Why throw this away?

Reply:
Currently, this is a second way of parsing for purpose of finding `// Error` assertions. That will go away, as those two way of doing the same thing will be unified. So this will not be needed anymore